string awb_mode
string exposure_mode
float32 red_gain
float32 blue_gain
uint16 iso
uint8 brightness
